README.md

```diff
@@ -92,7 +86,7 @@ In a terminal window run these commands if you need to update a snapshot.
 
 ```sh
 $ cd vets-who-code-app
-$ npm run test --updateSnapshot
+$ npm run test -- --updateSnapshot
```

Comment:
`-- --updateSnapshot`?

Reply:
Need to pass the command correctly to jest because this is run via npm.